piet-common/src/lib.rs

```diff
@@ -31,6 +32,10 @@ pub use piet::*;
 #[doc(hidden)]
 pub use piet::kurbo;
 
+#[doc(hidden)]
+#[cfg(feature = "image")]
+pub use piet::image_crate;
+
```

Comment:
I see this is following the example of kurbo above, but why have the re-exports be hidden? IIUC the point of the re-export is so that downstream crates can easily use the re-export instead of bringing in their own version, but then don't we want downstream users to know that the re-export is available?

Reply:
I agree with your thinking and thought the same. The reason that I ended up leaving it was that not having the `#[doc(hidden)]` actually causes it to be documented twice. This removes it from the "Re-exports" section, but leaves it in the modules section which is two lines below.

Now that you brought this up again I looked more closely, and it seems that it's not just the documentation that is double. The actual export seems to be redundant too.

```rust
pub use piet::*; // This line exposes piet::image (and kurbo) as modules.
```

I now removed these lines are everything that depends on them seems to keep compiling.